plugin/src/main/java/net/dzikoysk/funnyguilds/listener/region/PlayerMove.java

```diff
@@ -51,8 +53,9 @@ public void onMove(PlayerMoveEvent event) {
             User user = userOption.get();
             UserCache cache = user.getCache();
 
-            Option<Region> regionOption = this.regionManager.findRegionAtLocation(to);
-            if (regionOption.isEmpty() && user.getCache().getEnter()) {
+            Option<Region> regionOptionTo = this.regionManager.findRegionAtLocation(to);
+            Option<Region> regionOptionFrom = this.regionManager.findRegionAtLocation(from);
+            if (!regionOptionTo.equals(regionOptionFrom) && user.getCache().getEnter()) {
```

Comment:
Nad ifem zrób linijkę wolną. Swoją drogą, to bym najpierw dał if sprawdzający `user.getCache().getEnter()`, a potem zagnieździł niżej `            Option<Region> regionOptionFrom = this.regionManager.findRegionAtLocation(from);` z porównaniem. Wtedy w przypadku gdy enter -> false, to bez sensu nie będziesz requestował szukania tego drugiego regionu